portal/common/static/walrus-sites-portal-register-sw.js

```diff
@@ -24,22 +24,16 @@ function main() {
             })
             .catch(handleError);
     } else {
-        displayErrorMessage(swNotSupportedNode());
+        const currentUrl = new URL(window.location.href);
+        console.warn("This browser does not yet support Walrus Sites 💔, redirecting to blob.store");
+        window.location.href = currentUrl.toString().replace('walrus.site', 'blob.store');
```

Comment:
Important: using `walrus.site` breaks the portal when hosting on `localhost`
Suggestion: use `getPortalUrl`?

Other suggestion: make `blob.store` a constant, maybe under `constants.ts`, so that it is easy to customize for partners.

Reply:
I think a better way to do this would be:

```ts
window.location.href = new URL(
  `${currentUrl.pathname}${currentUrl.search}${currentUrl.hash}`,
  'https://blob.store',
);
```

this is more correct than basic string replacement.

Reply:
> I think a better way to do this would be:
> 
> ```ts
> window.location.href = new URL(
>   `${currentUrl.pathname}${currentUrl.search}${currentUrl.hash}`,
>   'https://blob.store',
> );
> ```
> 
> this is more correct than basic string replacement.

The problem with this solution is that the subdomain is not parsed. If we need to access docs.walrus.site, then the base of the url should be docs.blob.store, not just blob.store.

Reply:
> Important: using `walrus.site` breaks the portal when hosting on `localhost` Suggestion: use `getPortalUrl`?

Importing `getPortalUrl` is tricky in this script because it's a js file, and `getPortalUrl` is part of a ts library. We would need to bundle (transpile to js) the whole common/lib library in order to import something from there. 

> Other suggestion: make `blob.store` a constant, maybe under `constants.ts`, so that it is easy to customize for partners.

Yes that could be an easy addition! 👍

Reply:
So I combined both of the recommendations and moved the `blob.store` to a `FALLBACK_PORTAL` constant. 

Let me know what you think 🙆‍♂️